twoopstracker/twoops/models.py

```diff
@@ -156,3 +156,22 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class AccountGroup(TimestampedModelMixin):
```

Comment:
Since this is group of `UserProfile`s, lets call it `Team` or `UserProfileGroup`